src/Middleware/Rewrite/src/RewriteOptionsExtensions.cs

```diff
@@ -128,6 +128,17 @@ public static RewriteOptions AddRedirectToWwwPermanent(this RewriteOptions optio
             return AddRedirectToWww(options, statusCode: StatusCodes.Status308PermanentRedirect);
         }
 
+        /// <summary>
+        /// Permanently redirects the request to the www subdomain if the request is non-www.
+        /// </summary>
+        /// <param name="options">The <see cref="RewriteOptions"/>.</param>
+        /// <param name="domains">List of domains on which the www. rule will be applied.</param>
```

Comment:
```suggestion
        /// <param name="domains">Limit the rule to apply only on the specified domain(s).</param>
```

Reply:
I like the first option.

Reply:
Cool! All occurences updated with the last commit: https://github.com/aspnet/AspNetCore/pull/9676/commits/2dabf0701ea7d47d60797f72d21b66d7bcb80861